React/CoreModules/RCTActionSheetManager.mm

```diff
@@ -61,15 +69,10 @@ + (BOOL)requiresMainQueueSetup
 {
   return NO;
 }
-#endif // [macOS]
 
 RCT_EXPORT_MODULE()
 
-#if !TARGET_OS_OSX // [macOS]
 @synthesize viewRegistry_DEPRECATED = _viewRegistry_DEPRECATED;
-#else // [macOS
-@synthesize bridge = _bridge;
```

Comment:
didn't @amgleitman put this to resolve some bug before?

Reply:
Normally you access the viewRegistry from the bridge. But the bridge is going away with the new arch, so my guess is this was an upstream refactor where `RCTActionSheetManager` has a direct reference to the view registry so you don't need a reference to the bridge. 

The view registry is used to look up what view to anchor the menu on. In other parts of this code, I updated the calls and things seem to work.

Reply:
i see you are trying to remove all instances like line 405?

Reply:
Yep, that matches what iOS is doing. I just copied the iOS logic there.